jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/sam/SamTemplateUtils.kt

```diff
@@ -64,7 +66,8 @@ object SamTemplateUtils {
                         keyValue("Handler", handler)
                         keyValue("CodeUri", codeUri)
                         keyValue("Runtime", runtime.toString())
-                        keyValue("Timeout", "900")
+                        keyValue("Timeout", timeout.toString())
```

Comment:
how does this work if we hit a break-point while debugging?

Reply:
I've manually tested running in debug mode by calling `context.getRemainingTimeInMillis()`. Sam local set the timeout to be 10 hours ignoring the actual timeout in the template. So it should be ok in debug mode.

Reply:
Does it only do this in debug mode? If it does it all the time what's the point of setting the value at all?

Reply:
It only does it in debug mode. I am waiting for the confirmation from the sam team.